tests/core/test_accounts.py

```diff
@@ -82,13 +95,20 @@ def test_eth_account_create_variation(acct):
     assert account1 != account2
 
 
-def test_eth_account_equality(acct):
-    key = b'a' * 32
-    acct1 = acct.privateKeyToAccount(key)
-    acct2 = acct.privateKeyToAccount(key)
+def test_eth_account_equality(acct, PRIVATE_BYTES):
+    acct1 = acct.privateKeyToAccount(PRIVATE_BYTES)
+    acct2 = acct.privateKeyToAccount(PRIVATE_BYTES)
     assert acct1 == acct2
 
 
+def test_eth_account_privateKeyToAccount_fromPrivateKey(acct, PRIVATE_BYTES, PRIVATE_KEY):
```

Comment:
I think that our preferred way to do this would be as a parametrized fixture, or a parametrized test.

```
@pytest.fixture(params=[bytes, eth_keys.PrivateKey])
def private_key(request):
    return request.param(PRIVATE_KEY_BYTES)

# or

@pytest.mark.parametrize(
    'private_key',
    (
        PRIVATE_KEY_AS_BYTES,
        PRIVATE_KEY_AS_OBJ,
    ),
)
def test_the_thing(private_key):
    ...
```

```

Reply:
Updated here: 3938255

Let me know if the use of those constants is okay or if we'd rather just define them in the parameterize list.